flu/flu.prepare.py

```diff
@@ -27,16 +27,24 @@ def collect_args():
     parser.add_argument('-v', '--viruses_per_month_seq', type = int, default = 0, help='number of viruses sampled per month (optional) (defaults: 90 (2y), 60 (3y), 24 (6y) or 12 (12y))')
     parser.add_argument('--sampling', default = 'even', type=str,
                         help='sample evenly over regions (even) (default), or prioritize one region (region name), otherwise sample randomly')
+    parser.add_argument('--time_interval', nargs=2, help="explicit time interval to use -- overrides resolutions"
+                                                                     " expects YYYY-MM-DD YYYY-MM-DD")
     parser.add_argument('--strains', help="a text file containing a list of strains (one per line) to prepare without filtering or subsampling")
     parser.add_argument('--sequences', nargs='+', help="FASTA file of virus sequences from fauna (e.g., flu_h3n2_ha.fasta)")
     parser.add_argument('--titers', help="tab-delimited file of titer strains and values from fauna (e.g., h3n2_hi_titers.tsv)")
+    parser.add_argument('--dataset', help="dataset specifier")
```

Comment:
What is this?

Reply:
I have been using `--dataset` to tell apart the many different types of builds for CDC. this could be `FRA_cell` or similar. Not sure how you are handling this for CDC site.

Reply:
I believe this is my version of `--auspice-prefix`

Reply:
I see. Thanks. I was getting the concept of `dataset` confused with `h3n2_3y`, etc... What if we make this thing be tied semantically to titer set? Will that cover the use cases? So like `titerset` for things like `fra_cell` and if necessary we end up with an additional `source` for `cdc` vs `crick`. Alternatively, we'd have a general slop term like `identifier` or something that would contain all of this, ie `cdc_fra_cell`.

Reply:
I'd rather have `identifier` or similar. I can see many cases where this is not titer but say `chicken` or `china`. and `prefix` is not good as it is rarely a prefix.

Reply:
Okay. I like this. So about this: 

We keep `--lineage`, `--resolution`, etc... for passing in parameters to process. We additionally have a `--identifier` parameter.

In one scenario `--identifier` is not set and we build one based off of `lineage` and `resolution`, so that `--lineage h3n2 --resolution 3y` results in `identifier = h3n2_3y` and JSONs that look like `h3n2_3_tree.json`.

In another scenario `--identifier` is set and we use it _instead_ of lineage and resolution,  so that `--lineage h3n2 --resolution 3y --identifier temp` results in  JSONs that look like `temp_tree.json`.